templates/default/mgm-server-start.sh.erb

```diff
@@ -23,11 +23,20 @@ if [ ! -e <%= node['mysql']['base_dir'] %>/bin/ndb_mgmd ] ; then
     exit 3
 fi
 
+INITIAL_START_ARG=
+if [ -n "$INITIAL_START" ]; then
+    INITIAL_START_ARG="--initial"
+fi
+
+SERVICE_ARG=
+if [ -n "$SERVICE_NAME" ]; then
+    SERVICE_ARG="--service-name $SERVICE_NAME"
+fi
 
 <% if #{node['ndb']['use_systemd']} == "true" -%>
-<%= node['mysql']['base_dir'] %>/bin/ndb_mgmd --ndb-nodeid=<%= @node_id %> -f <%= node['ndb']['root_dir'] %>/config.ini  --configdir=<%= node['ndb']['mgm_dir'] %> --reload --initial
+<%= node['mysql']['base_dir'] %>/bin/ndb_mgmd --ndb-nodeid=<%= @node_id %> -f <%= node['ndb']['root_dir'] %>/config.ini  --configdir=<%= node['ndb']['mgm_dir'] %> --reload ${INITIAL_START_ARG} ${SERVICE_ARG}
 <% else -%>
-su <%= node['ndb']['user'] %> -c \"<% node['mysql']['base_dir'] %>/bin/ndb_mgmd --ndb-nodeid=<%= @node_id %> -f <%= node['ndb']['root_dir'] %>/config.ini  --configdir=<%= node['ndb']['mgm_dir'] %> --reload --initial\"
+su <%= node['ndb']['user'] %> -c \"<% node['mysql']['base_dir'] %>/bin/ndb_mgmd --ndb-nodeid=<%= @node_id %> -f <%= node['ndb']['root_dir'] %>/config.ini  --configdir=<%= node['ndb']['mgm_dir'] %> --reload ${INITIAL_START_ARG} ${SERVICE_ARG}\"
```

Comment:
do we still need this here? since you remove the node_id variable this will not work, i mean in general we always use systemd but just for cleaning up the code
`--ndb-nodeid=<%= @node_id %>`

Reply:
Not sure I follow, why would this not be needed, how would we pass the initial start argument and service argument

Reply:
I meant `@node_id` is not passed anymore to this file and this else condition is never used since we always use systemd but we should clean it up regardless 
```
<% if #{node['ndb']['use_systemd']} == "true" -%>
<%= node['mysql']['base_dir'] %>/bin/ndb_mgmd --ndb-nodeid=<%= @node_id %> -f <%= node['ndb']['root_dir'] %>/config.ini  --configdir=<%= node['ndb']['mgm_dir'] %> --reload ${INITIAL_START_ARG} ${SERVICE_ARG}
<% else -%>
su <%= node['ndb']['user'] %> -c \"<% node['mysql']['base_dir'] %>/bin/ndb_mgmd --ndb-nodeid=<%= @node_id %> -f <%= node['ndb']['root_dir'] %>/config.ini  --configdir=<%= node['ndb']['mgm_dir'] %> --reload ${INITIAL_START_ARG} ${SERVICE_ARG}\"
<% end -%>
```